docs/antctl.md

```diff
@@ -235,13 +235,15 @@ antctl get podinterface [NAME] [-n NAMESPACE]
 
 Starting from version 0.6.0, Antrea Agent supports dumping Antrea OVS flows. The
 `antctl` `get ovsflows` (or `get of`) command can dump all OVS flows, flows
-added for a specified Pod, or flows added to realize a specified NetworkPolicy,
-or flows in a specified OVS flow table.
+added for a specified Pod, or flows added for Service load-balancing of a
+specified Service, or flows added to realize a specified NetworkPolicy, or flows
+in a specified OVS flow table.
 
 ```bash
 antctl get ovsflows
 antctl get ovsflows -p POD -n NAMESPACE
-antctl get ovsflows --networkpolicy NETWORKPOLICY -n NAMESPACE
+antctl get ovsflows -S SERVICE -n NAMESPACE
```

Comment:
Not related to this commit.
Why we use `p` for Pod but capitalized `S` for others?

Reply:
-s (--server) is a common flag of antctl, for specifying a server address.

Reply:
Got it. Thanks.